Documentation/WSL.md

```diff
@@ -0,0 +1,8 @@
+### **Building on Windows, Targeting Linux via WSL**
```

Comment:
nit: Consider
```suggestion
# Building on Windows, Targeting Linux via WSL
```
Since this file is just for WSL, it can be the title instead of a tier 3 heading if you want.